workers.go

```diff
@@ -313,13 +313,6 @@ func (wm *WorkerManager) triggerShutdownIfRequired(decision *FailedDecision) {
 	}
 }
 
-func (wm *WorkerManager) stopBatch() {
```

Comment:
This is related to #121 - the failure handling logic was broken and we just forgot to remove that `stopBatch`. Thanks for noticing